Makefile

```diff
@@ -12,18 +12,7 @@
 #See the License for the specific language governing permissions and
 #limitations under the License.
 
-.PHONY: manifests \
-		cluster-up cluster-down cluster-sync \
-		test test-functional test-unit test-lint \
-		publish \
-		vet \
-		format \
-		goveralls \
-		coverage \
-		release-description \
-		bazel-generate bazel-build bazel-build-images bazel-push-images \
-		fossa \
-		lint-metrics
+.PHONY: help all clean update-codegen generate bootstrap-ginkgo generate-verify gomod-update apidocs deps-update deps-verify rpm-deps build-functest test test-unit test-unit test-functional test-functional goveralls coverage docker-registry-cleanup publish manifests release-description builder-push openshift-ci-image-push cluster-up cluster-down cluster-down-purge cluster-sync-cdi cluster-sync-test-infra cluster-sync bazel-generate bazel-build bazel-push-images push build-docgen generate-doc fossa lint-metrics test-lint vet vulncheck format
```

Comment:
Yeah, I don't really like either option because this is inevitably gonna go stale as soon as any target is added/removed/modified.

I was thinking maybe this:
```Makefile
.PHONY: target1
target1: ## Target one is the first target
    do_thing

.PHONY: target2
target2: target1 ## Target two is the second target
    do_other_thing

.PHONY: target3
target3: target1 target2 ## Target three is the third target
    do_something_else
```
It's a bit more verbose, but it seems more likely to stay up to date if it's near the target.

What do you think?

Reply:
Im not sure if its so critical.. first its not like we add a target everyday :) second if someone will add a file with the same name of a target that someone forgot to put phony on it it will not perform as expected and it will be pretty simple to understand.. do what you think is best, just saying that in any other kubevirt project I just saw that phony is in a long list either at the start or the end of the Makefile

Reply:
Ok, that makes sense. I'll move it to the end as you suggested.